### ISSUE TITLE
Can't train model, only green samples generated

### ISSUE THREAD

**Issue by furkan-celik** [CompVis/stable-diffusion#589]
Hi,

I am working on a project where we want to use Stable Diffusion. I have already used OpenAI's improved-diffusion and score-guided diffusion on my environment. However, when I try to use stable diffusion all I get is green samples and blank picture for class conditioned images. I have looked at this and it is the same from the initialization of the model but no weight is NA or INF. When I use extreme learning rates outputs get black but still no learning. Loss also doesn't change and stay at 1. Any idea how to resolve this?

GPU: Nvidia A40
Image size: 256x256
I have tried with CelebA and custom dataset with ImageFolder function from torchvision.
Environment is set by 'conda env create -f environment.yaml' and environment is clean besides these packages.
Tried running for 12 hours observed no change on outputs.
Inputs saved from ImageLogger looks fine.
This is also mentioned in #69 but no solution is provided.

Reconstruction result
![reconstruction_gs-000003_e-000000_b-000003](https://user-images.githubusercontent.com/6495679/214249738-ae6fc7e1-057e-4213-8207-73e026fb5771.png)
Diffusion Row Result
![diffusion_row_gs-000000_e-000000_b-000000](https://user-images.githubusercontent.com/6495679/214249807-e2724881-0903-4d65-9888-78b8cf94e841.png)

**Comment by fujistoo** [comment]
Did you manage to solve this?

**Comment by furkan-celik** [comment]
Yes, I can't recall how I solved thıs though. Do you still need assistance?
